tests/test_i18n.py

```diff
@@ -55,8 +59,12 @@ def test_changed_entities(self):
         tx = self.session.query(tx_log).order_by(sa.desc(tx_log.id)).first()
         assert "ArticleTranslation" in tx.entity_names
 
+    @mark.skipif(
+        "os.environ.get('DB', 'sqlite') == 'oracle'",
+        reason="tranaction.changes when queried with oracle DB returns list in arbitary order.",
+    )
     def test_history_with_many_translations(self):
-        self.article = self.Article()
+        self.article = self.Article(id=1)
```

Comment:
Oracle is non deterministic i how the relationships are queried.
So, we need to use: `assert "ArticleTranslation" in {i.entity_name for i in transaction.changes}`